src/util/metadata/side_metadata/global.rs

```diff
@@ -178,31 +224,21 @@ impl SideMetadataSpec {
                 - start.align_down(BYTES_IN_CHUNK))
                 / BYTES_IN_CHUNK;
             if chunk_num == 0 {
-                memory::zero(
-                    meta_start,
-                    address_to_meta_address(self, start + size) - meta_start,
-                );
+                zero(start, start + size);
             } else {
                 let second_data_chunk = start.align_up(BYTES_IN_CHUNK);
                 // bzero the first sub-chunk
-                memory::zero(
-                    meta_start,
-                    address_to_meta_address(self, second_data_chunk) - meta_start,
-                );
+                zero(start, second_data_chunk);
+
                 let last_data_chunk = (start + size).align_down(BYTES_IN_CHUNK);
                 let last_meta_chunk = address_to_meta_address(self, last_data_chunk);
                 // bzero the last sub-chunk
-                memory::zero(
-                    last_meta_chunk,
-                    address_to_meta_address(self, start + size) - last_meta_chunk,
-                );
+                zero(last_meta_chunk, start + size);
```

Comment:
The `zero` function takes data address as parameter, but it passes `last_meta_chunk`.  It should be `last_data_chunk`.
```suggestion
                zero(last_data_chunk, start + size);
```

Reply:
Thanks. Fixed.